src/components/ui/AudioPlayer.tsx

```diff
@@ -17,43 +17,52 @@ import {
 import { Panel } from "components/ui/Panel";
 import { useStepper } from "lib/utils/hooks/useStepper";
 
-const LOCAL_STORAGE_KEY = "settings.audioMuted";
-
-function cacheAudioMuted(muted: boolean) {
-  localStorage.setItem(LOCAL_STORAGE_KEY, JSON.stringify(muted));
+enum AudioLocalStorageKeys {
+  audioMuted = "settings.audioMuted",
+  musicPaused = "settings.musicPaused",
+  musicVolume = "settings.musicVolume",
 }
 
-function getCachedAudioMuted(): boolean {
-  const cached = localStorage.getItem(LOCAL_STORAGE_KEY);
+function cacheAudioSetting(
+  key: AudioLocalStorageKeys,
+  value: boolean | number
+) {
+  localStorage.setItem(key, JSON.stringify(value));
+}
 
-  if (!cached) {
-    return false;
-  }
+function getCachedAudioSetting<T>(
+  key: AudioLocalStorageKeys,
+  defaultValue: T
+): T {
+  const cached = localStorage.getItem(key);
 
-  return JSON.parse(cached);
+  return cached ? JSON.parse(cached) : defaultValue;
 }
 
 interface Props {
   isFarming?: boolean;
 }
 
 export const AudioPlayer: React.FC<Props> = ({ isFarming }) => {
-  const volume = useStepper({ initial: 0.1, step: 0.1, max: 1, min: 0 });
+  const volume = useStepper({
+    initial: getCachedAudioSetting<number>(
+      AudioLocalStorageKeys.musicVolume,
+      0.1
+    ),
+    step: 0.1,
+    max: 1,
+    min: 0,
+  });
   const [visible, setIsVisible] = useState<boolean>(false);
-  const [muted, setMuted] = useState<boolean>(getCachedAudioMuted());
-  const [isPlaying, setPlaying] = useState<boolean>(true);
+  const [audioMmuted, setAudioMuted] = useState<boolean>(
+    getCachedAudioSetting<boolean>(AudioLocalStorageKeys.audioMuted, false)
+  );
+  const [musicPaused, setMusicPaused] = useState<boolean>(
+    getCachedAudioSetting<boolean>(AudioLocalStorageKeys.musicPaused, true)
```

Comment:
Could we make it playing by default instead of paused?

Reply:
Oh, good catch, intention was to autoplay music on non chromium browsers, but It looks like I forgot that value when refactoring from isPlaying to musicPaused. Fixed. :)